From b30c27adebd9f97eb7f4a835d354ac8b92a055cd Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 10 Nov 2020 13:59:31 +0800
Subject: [PATCH] arm64: dts: rockchip: set the bootargs with rw for rk356x
 linux

The Linux rootfs allow to be readable and writable by default.

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: Ibc8de371c2b27a5062311e079bfe3389c5ffd6c8
---
 arch/arm64/boot/dts/rockchip/rk3568-linux.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-linux.dtsi
index be667b7be755..c9f05c85bc69 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-linux.dtsi
@@ -6,7 +6,7 @@
 
 / {
 	chosen: chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xfe660000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xfe660000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rw rootwait";
 	};
 
 	fiq-debugger {
-- 
2.35.3

